zcallback_darwin_amd64.s

```diff
@@ -0,0 +1,2014 @@
+// Code generated by wincallback.go using 'go generate'. DO NOT EDIT.
```

Comment:
The zcallback files are copied from runtime but they are generated by a file. I also changed them to call purego’s version of callbackasm1 instead of the runtime version. There is no license header to these files since they are generated

Reply:
The generator is also in the runtime package